apps/files/lib/Command/TransferOwnership.php

```diff
@@ -151,6 +160,16 @@ protected function execute(InputInterface $input, OutputInterface $output) {
 		$this->inputPath = $input->getOption('path');
 		$this->inputPath = \ltrim($this->inputPath, '/');
 
+		$this->destinationPath = $input->getOption('destination-path');
+		if ($this->destinationPath === null) {
+			\OC::$server->getLogger()->debug('When destinationPath not defined we will use the default');
```

Comment:
```suggestion
			\OC::$server->getLogger()->debug('The destinationPath is not defined, using the default one');
```
based on the query, it should result in a statement not a question.

Reply:
I don't think we need this log. Same with the log below.